### PR TITLE
draft: add closeAndGetPendingLocalState() coverage to stress test

### PR DESCRIPTION
**Pull request by wes-carlson**
## Description

Add coverage of Container.closeAndGetPendingLocalState() and SharedObject.applyStashedOp() flow to stress test.

#AB123

## Reviewer Guidance

See design doc

### CODE REVIEW

File: packages/loader/container-loader/src/container.ts [hunk 1]
```diff
@@ -631,12 +631,15 @@ export class Container extends EventEmitterWithErrorHandling<IContainerEvents> i
...
+        // TODO: this information should be stored in the blob since we may have a blob with no pending changes
+        this._dirtyContainer = config.serializedContainerState !== undefined;
+
         this.connectionStateHandler = new ConnectionStateHandler(
             {
                 quorumClients: () => this._protocolHandler?.quorum,
                 logConnectionStateChangeTelemetry: (value, oldState, reason) =>
                     this.logConnectionStateChangeTelemetry(value, oldState, reason),
-                shouldClientJoinWrite: () => this._deltaManager.connectionManager.shouldJoinWrite(),
+                shouldClientJoinWrite: () => this.isDirty,
```
Comment (curtisman) [inline packages/loader/container-loader/src/container.ts:642]: I don't know whether you have talked to @vladsud or @kian-thompson about this change, but I believe they are doing some change in this area.  Please make sure that this aligns with their plan.

File: packages/test/test-service-load/src/runner.ts [hunk 2]
```diff
@@ -242,14 +283,80 @@ async function runnerProcess(
...
+    const ds = dsf.documentServices.get(container.resolvedUrl);
+    assert(!!ds, "no documentServices");
+
+    printStatus(runConfig, `going offline! closing in ${offlineTime / 1000} seconds!`);
+    ds.goOffline();
+
+    await new Promise<void>((res) => setTimeout(res, offlineTime));
+    if (!container.closed) {
+        printStatus(runConfig, "stashing changes!");
+        return container.closeAndGetPendingLocalState();
```
Comment (curtisman) [inline packages/test/test-service-load/src/runner.ts:356]: closeAndGetPendingLocalState could be called even if it is not offline, right?

File: packages/loader/container-loader/src/container.ts (hunk 1 above)
Comment (kian-thompson) [reply packages/loader/container-loader/src/container.ts:642]: I don't think this changes anything on our side, but I'll update if it does

File: packages/test/test-service-load/src/runner.ts (hunk 2 above)
Comment (wes-carlson) [reply packages/test/test-service-load/src/runner.ts:356]: Yes, but we would only stash what happens to be pending at that time.

**Pull request closed without merging**